packages/iTwinUI-react/src/core/DropdownMenu/DropdownMenu.test.tsx

```diff
@@ -19,7 +19,10 @@ function assertBaseElement(menu: HTMLUListElement, role = 'menu') {
   });
 }
 
-function renderComponent(props?: Partial<DropdownMenuProps>) {
+function renderComponent(
```

Comment:
I don't understand why test `should render filter dropdown in the correct document` in Table fails. So I tried to do the same in other tests and it fails here too. Anyone has any suggestions?